dockerfile/security/secret-in-build-arg.dockerfile

```diff
@@ -0,0 +1,33 @@
+# ok: secret-in-build-arg
+ARG BASE_IMG=rust   # default image
```

Comment:
I don't think you can have comments on the same line of the ARG command (I see a syntax error when running this in the playground).
I would simply these first four lines to:
```
# default image
# ok: secret-in-build-arg
ARG BASE_IMG=rust   
# default tag
# ok: secret-in-build-arg
ARG BASE_TAG=latest
```